compute/admin_guide/configure/custom_feeds.adoc

```diff
@@ -78,36 +78,39 @@ It is also used immediately by the runtime defense file system sensor, which ass
 [.task]
 === Create a list of trusted executables
 
-The Trusted Executable Feed is a mechanism to address potential misidentification of legitimate files as malware. You can add to the feed signatures of binaries that have been incorrectly categorized as malicious by the file system based malware runtime protection. 
+The trusted executable list is a mechanism to address potential misidentification of legitimate files as malware. When you add signatures of binaries to this list, it enables you to ensure that a legitimate binary or a process created from a legitimate binary is not potentially identified as malicious by runtime defense rules that check for file system based malware. 
```

Comment:
1. Remove this part, it is incorrect: `or a process created from a legitimate binary`
2. Suggest to change this:
```
identified as malicious by runtime defense rules that check for file system based malware.
```
to this:
```
identified as malicious by the runtime rules file system based defense capability.
```

Reply:
done. broke it up to avoid  a long string of nouns.